.github/workflows/kythe.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Install Kythe, extract Kzips, and upload to GCP to enable semantic indexing on
+# https://cs.opensource.google/opentitan
+
+name: kythe
+
+on:
+  # For debugging, it's useful to have the action run whenever the PR is
+  # modified. This can be accomplished by setting a bare `pull_request:` with no
+  # items specified.
+  #
+  # Under normal circumstances, the action should only run when PRs are merged,
+  # and possibly less frequently.
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges-1
+  #
+  # TODO(dmcardle) Run only when PRs are merged.
+  pull_request:
```

Comment:
Are you planning to remove these before submitting? I assume that you are still working on getting this to turn green in your presubmit?